content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,155 @@
+  settings using configuration parameters. If these settings are available, you
+  should proceed with caution before modifying the default configuration values.
+  Improperly setting these values might expose the OpenTelemetry Collector to
+  additional attack vectors including resource exhaustion.
+
+### Safeguards against denial of service attacks
+
+Bind receivers' servers to addresses that limit connections to authorized users,
+so that your Collectors aren't exposed to the public internet or to wider
+networks than necessary.
+
+For example, if the OTLP receiver OTLP/gRPC server has only local clients, bind
+the `endpoint` setting to `localhost`:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
+```
+
+Use `localhost`-like addresses instead of `0.0.0.0`. For more information, see
+[CWE-1327: Binding to an Unrestricted IP Address](https://cwe.mitre.org/data/definitions/1327.html).
+
+To change the default endpoint to be `localhost`-bound in all components, enable
+the `component.UseLocalHostAsDefaultHost` feature gate. This feature gate will
+be enabled by default in the Collector in a future release.
+
+### Encryption and authentication
+
+Your OTel Collector configuration should include encryption and authentication.
+
+- For communication encryption, see
+  [Configuring certificates](/collector/configuration/#setting-up-certificates).
+- For authentication, use the OTel Collector's authentication mechanism, as
+  described in [Authentication](/collector/configuration/#authentication).
+
+## Processors
+
+[Processors](/collector/configuration/#processors) sit between receivers and
+exporters. They are responsible for processing telemetry before it's analyzed.
+From a security perspective, processors are useful in a few ways.
+
+### Recommended processors
+
+<!--- TODO: SHOULD configure recommended processors. If so, what are they?
+
+redaction processor (never tried myself)
+transform processor (great for redacting PIIs and such) -->
+
+### Scrubbing sensitive data
+
+You can use the OpenTelemetry Collector to scrub sensitive data before exporting
+it to a backend. Configure the Collector to obfuscate or scrub sensitive data
+before exporting.
+
+<!--- TODO: SHOULD configure obfuscation/scrubbing of sensitive metadata. How? Give more details and/or link to an existing document -->
+
+Use OpenTelemetry Collector's `redaction` processor to scrub sensitive data.
+
+<!--- TODO: Give example config for the redaction processor or remove this line. --->
+
+### Safeguarding resource utilization
+
+Processors also offer safeguards for resource utilization. After implementing
+safeguards for resource utilization in your
+[hosting infrastructure](/security/hosting-best-practices/), make sure your
+OpenTelemetry Collector configuration uses these safeguards.
+
+<!-- start same page content in hosting-best-practices -->
+
+The `batch` and `memory_limiter` processors help ensure the OpenTelemetry
```

Comment:
Given the discussions we are having around the batch processor, we should probably stop advertising it.

cc @mx-psi

Reply:
After a second thought, perhaps we should stop talking about the memory limiter _processor_ as well, mentioning the _extension_ instead once it gets ready?

https://github.com/open-telemetry/opentelemetry-collector/tree/main/extension/memorylimiterextension

Reply:
Based on @reyang's earlier comment that resource utilization is an important topic, it seems like we should say something about best practices here. If we don't recommend the batch and memory limiter processors, and the memory limiter extension is still in development, how should we advise users to safeguard resource utilization in their Collector configurations and hosting?

Reply:
I took a shot at writing this section on my own (8605abb), but it could definitely use a developer's review. 

I mention batching and memory limiting, but I don't call out the processors or extensions specifically. Once the memory limiter extension is ready, we can update with an example.

Reply:
I think a general mention to batching and memory limiting should be more future-proof